doc/model/model.rst

```diff
@@ -21,36 +24,230 @@ Model
+conductive and diffusive components, $\mathbf{q}_{c}$ and $\mathbf{q}_{d}$,
+respectively:
+
+.. math::
+   \mathbf{q} = \mathbf{q}_c + \mathbf{q}_d
+
+
+Conductive heat flux
+^^^^^^^^^^^^^^^^^^^^
+The conductive heat flux vector is defined as
+
+.. math::
+    \mathbf{q}_c = -\kappa\nabla{T},
+
+where $\kappa$ is the thermal conductivity, and ${T}$ is the gas
+temperature.
+
+Diffusive heat flux
+^^^^^^^^^^^^^^^^^^^
+The diffusive heat flux vector is defined as
+
+.. math::
+   \mathbf{q}_d = {h}_{\alpha}\mathbf{J}_{\alpha},
+
+with the species specific enthalpy ${h}_{\alpha}$, and the species
+diffusive flux vector $\mathbf{J}_{\alpha}$.
+
+.. _Chemistry:
+
+Chemistry
+---------
+
+Chemical reactions introduce source terms in the energy and species conservation equations.
+The species source term is the amount of mass produced for each species:
+
+.. math::
+   W^{\mathtt{chem}}_{\alpha} = w_{(\alpha)}\dot{\omega}_{\alpha},
+
+where ${w}_{\alpha}$ is the molecular weight of each species, and $\dot{\omega}_{\alpha}$ is the net
+chemical production rate for each species. Here, the parens $(\alpha)$ indicates no sum is to be performed
+over repeated indices. 
+
+The energy source term is the amount of thermal energy used to create each species:
+
+.. math::
+   E^{\mathtt{chem}} = -h^f_{\alpha}W^{\mathtt{chem}}_{\alpha},
+
+where $h^f_{\alpha}$ is the enthalpy of formation for each species.
+
+.. _eos-and-matprop:
+
+Equations of State and Material properties
+------------------------------------------
+
+Equations of state (EOS) provide functions that relate the fluid state $Q$, and the
+thermodynamic properties such as pressure $p$, temperature $T$, specific enthalpies $h_{\alpha}$,
+and total energy $E$.  The EOS provided *MIRGE-Com* are documented in :mod:`mirgecom.eos`.
+
+Material properties including the first coefficient of viscosity, $\mu$, bulk viscosity $\mu_B$,
+thermal conductivity $\kappa$, and species diffusivities ${d}_{\alpha}$ depend on the state of
+the fluid $\mathbf{Q}$, in general, and are provided by transport models.  Transport models provided
+by *MIRGE-Com* ~~are~~ (will be) documented in forthcoming the transport module.
+
+
+.. _disc-strat:
+
+Discretization Strategy
+-----------------------
+
+How to discretize the conservation equations with DG, including how to handle the required fluxes,
+particularly in the viscous setting, is a current topic of research and internal discussion.  The
+following references are useful:
+
+* "The DG Book:" Nodal Discontinuous Galerkin Methods, [Hesthaven_2008]_
+* The BR1 algorithm for discretization of Navier-Stokes, [Bassi_1997]_
+* NS with reactions, [Ihme_2014]_, and [Cook_2009]_
+* The BR2 algorithm, [Bassi_2000]_
+* [Ayuso_2009]_
+
+*MIRGE-Com* currently employs a strategy akin to the BR1 algorithm outlined in [Bassi_1997]_, but
+with thermal terms and chemical reaction sources as outlined in [Ihme_2014]_ and [Cook_2009]_.
+
```

Comment:
Would it be worth quickly summarizing the DG strategy as we've done [here](https://notes.tiker.net/vFDwHwNhQTyTZYTFMI-7lw?both)? I realize there's still work to be done on the boundary conditions side of things, but I think having a description of the DG problem is very useful to see. What do you think?

Reply:
Love the idea.  Should we start a discretization page and just link it here?

Reply:
I also think it should be extended to include the thermal pieces, chemistry, and mixtures.  I guess that could come later though.

Reply:
(e0c424f)